image_view/src/nodes/video_recorder.cpp

```diff
@@ -70,13 +71,20 @@ void callback(const sensor_msgs::ImageConstPtr& image_msg, const sensor_msgs::Ca
     }
     else if (outputVideo.isOpened() && info) return;
 
+    if ((g_last_wrote_time != ros::Time(0)) && ((ros::Time::now() - g_last_wrote_time) < ros::Duration(1 / fps)))
```

Comment:
instead of ros::Time::now(), you should use image_msg->header.stamp

The first condition is probably useless right ?